src/ci/agent.sh

```diff
@@ -38,13 +38,10 @@ cargo build --release --locked
 cargo clippy --release -- -D warnings
 # export RUST_LOG=trace
 export RUST_BACKTRACE=full
-cargo test --release --manifest-path ./onefuzz-supervisor/Cargo.toml
+cargo test --release --workspace
```

Comment:
This seems nicer; does it enable any other tests that we weren't running previously?

Just ICYMI: cc @bmc-msft, @chkeita

Reply:
It does, one of the changes in this PR fixes one of the tests that were not run.